sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AppConfigurationExporterIntegrationTest.java

```diff
@@ -41,6 +41,7 @@ public class AppConfigurationExporterIntegrationTest extends MonitorExporterClie
     @BeforeEach
     public void setupTest(TestInfo testInfo) {
         Assumptions.assumeFalse(getTestMode() == TestMode.PLAYBACK, "Skipping playback tests");
+        super.setupTest(testInfo);
```

Comment:
why do we need this?

Reply:
This is needed for the setup to be done (i.e. call the setupTest method of the base class) when the test mode is not PLAYBACK.